java-manta-client/src/main/java/com/joyent/manta/client/MantaClient.java

```diff
@@ -205,7 +206,7 @@ public MantaClient(final ConfigContext config) {
         final KeyPair keyPair = keyPairFactory.createKeyPair();
 
         final Signer.Builder builder = new Signer.Builder(keyPair);
-        if (config.disableNativeSignatures()) {
+        if (ObjectUtils.firstNonNull(config.disableNativeSignatures(), false)) {
```

Comment:
I suggest that you add a constant field to `DefaultsConfigContext`, so that you reference the default value rather than explicitly set it here.